src/request.ts

```diff
@@ -71,9 +72,15 @@ class Request {
     return this.execute(options);
   }
 
-  public get<TResponse>(options: AxiosRequestConfig): Promise<TResponse> {
+  public get<TResponse>(options: AxiosRequestConfig, isLargeFile?: boolean): Promise<TResponse> {
```

Comment:
How about we keep this method intact and let the `responseType` be set along with other options where the request is configured? That way we keep this method generic and applicable to all different use cases rather than have it implement specific use cases.